config/cm-libraries.conf

```diff
@@ -0,0 +1,2 @@
+xyzlib|http://simplye-dev-cm.amigos.org/xyzlib
```

Comment:
Was this file supposed to be include in the PR? I see it's also in the .gitignore, but it is referenced in the readme.

Reply:
No, it wasn't. I was playing around with possibilities and forgot the gitignore initially. Looks like this crept in before. It's not a big deal--it's just a demo system I've got up for development. But I should take it out before it's merged. I'll do that in a bit. But see comment below on what to include.